src/observable/dom/AjaxObservable.ts

```diff
@@ -191,7 +191,7 @@ export class AjaxSubscriber<T> extends Subscriber<Event> {
     }
 
     // ensure content type is set
-    if (!('Content-Type' in headers) && !(root.FormData && request.body instanceof root.FormData)) {
+    if (!('Content-Type' in headers) && !(root.FormData && request.body instanceof root.FormData) && !(typeof request.body === 'undefined')) {
```

Comment:
`typeof request.body !== 'undefined'` please. The reason the others have it outside is because it's the only way to negate `in` or `instanceof`.